code/parachain/frame/pablo/build.rs

```diff
@@ -0,0 +1,197 @@
+#![allow(clippy::disallowed_methods)] // temporarilly
+
+use std::{env, fmt::Write, fs, iter, path::PathBuf};
+
+use syn::{Generics, Item, ItemFn, ItemMod, ReturnType, Signature, Visibility};
+
+fn main() {
+	aggregate_tests("testing");
+}
```

Comment:
Please sanity check that nix does run this and that these test work under nix :)